src/controls/Controls.spec.js

```diff
@@ -1 +1,37 @@
 // Test away!
+import React from "react";
+import { render, fireEvent } from "react-testing-library";
+import "react-testing-library/cleanup-after-each";
+import renderer from "react-test-renderer";
+import "jest-dom/extend-expect";
+import Controls from "./Controls";
+
+describe("Controls Module", () => {
+  it("should render on load", () => {
+    render(<Controls />);
+  });
+
+  it("should display lock gate text on the button if locked is false and closed is true", () => {
+    const locked = false;
+    const closed = true;
+
+    const { getByText } = render(<Controls locked={locked} closed={closed} />);
+    const text = getByText(/lock gate/i);
+    expect(text).toBeInTheDocument();
+  });
+
+  it("should display open gate text on the button if locked is false and closed is true", () => {
+    const locked = false;
+    const closed = true;
+
+    const { getByText } = render(<Controls locked={locked} closed={closed} />);
+    const text = getByText(/open gate/i);
+    expect(text).toBeInTheDocument();
```

Comment:
You could write a helper for these two similar test callbacks:
```javascript
const makeControlsTest = ({ locked, closed, text }) => {
  return () => {
    const { getByText } = render(<Controls locked={locked} closed={closed} />);
    text = getByText(text);
    expect(text).toBeInTheDocument();
  };
};
```
and the usage, using this third test as an example:
```javascript
it("should display open gate text on the button if locked is false and closed is true", 
  makeControlsTest({ locked: false, closed: true, text: /open gate/i })
);
```